oracle/types/market.go

```diff
@@ -53,14 +53,34 @@ func ProviderTickersFromMarketMap(
 type TickersToProviderTickers map[mmtypes.Ticker]DefaultProviderTicker
 
 // ToProviderTickers converts the map to a list of provider tickers.
-func (tpt *TickersToProviderTickers) ToProviderTickers() []ProviderTicker {
-	var providerTickers = make([]ProviderTicker, len(*tpt))
+func (tpt TickersToProviderTickers) ToProviderTickers() []ProviderTicker {
+	var providerTickers = make([]ProviderTicker, len(tpt))
 
 	i := 0
-	for _, ticker := range *tpt {
+	for _, ticker := range tpt {
+		// If the ticker does not have a set number of decimals, set it to the default.
+		if ticker.Decimals == 0 {
+			ticker.Decimals = DefaultTickerDecimals
+		}
+
 		providerTickers[i] = ticker
 		i++
 	}
 
 	return providerTickers
 }
+
+// MustGetProviderTicker returns the provider ticker for the given x/marketmap ticker.
+// This function is mostly used for testing.
+func (tpt TickersToProviderTickers) MustGetProviderTicker(ticker mmtypes.Ticker) ProviderTicker {
+	providerTicker, ok := (tpt)[ticker]
+	if !ok {
+		panic(fmt.Sprintf("ticker %s not found", ticker))
```

Comment:
Kind of seems like a footgun having this panic.

Reply:
I only use this in testing so it's probably fine?

Reply:
`Must` prefices are p standard in cosmos, and they always panic instead of error. If we only use in tests, can we just move this to testing, and have it wrap the existing getter?